packages/urls/src/urls.ts

```diff
@@ -3,6 +3,7 @@ export const TREZOR_FORUM_URL = 'https://forum.trezor.io/';
 export const TREZOR_START_URL = 'https://trezor.io/start';
 export const TREZOR_SUPPORT_URL = 'https://trezor.io/support';
 export const TREZOR_RESELLERS_URL = 'https://trezor.io/resellers/';
+export const TREZOR_TRANSACTION_FEES_SUITE = 'https://trezor.io/learn/a/transaction-fees-in-trezor-suite';
```

Comment:
Every URL in this file ends with URL 🙏
And there are urls going to learn, so put it there

Reply:
Fixed